ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1676,10 +1713,7 @@ public string GetServerArgs()
                 serverArgs.Append("?ForceAllStructureLocking=true");
             }
 
-            if (this.AutoDestroyOldStructuresMultiplier >= 1.0f)
-            {
-                serverArgs.AppendFormat("?AutoDestroyOldStructuresMultiplier={0}", AutoDestroyOldStructuresMultiplier);
-            }
+            serverArgs.AppendFormat("?AutoDestroyOldStructuresMultiplier={0}", AutoDestroyOldStructuresMultiplier);
```

Comment:
Consider string interpolation:
.Append($"?AutoDestroyOldStructuresMultiplier={AutoDestroyOldStructuresMultiplier}") 

I find it slightly easier to read than String.Format() style composition.